### ISSUE TITLE
balances UI ugly

### ISSUE THREAD

**Issue by aupiff** [blockmason/lndr-mobile#41]
<img width="364" alt="screen shot 2017-11-22 at 10 03 06" src="https://user-images.githubusercontent.com/804266/33134045-a73874fe-cf6c-11e7-9b9b-bf10738de90e.png">
weird alignment issues.

**Comment by mattsmithies** [comment]
Yes I see, perhaps we could have left, center, right alignment with the address comprising of the first/last eight characters.

**Comment by aupiff** [comment]
the balance color is off here as well, negative numbers should be in red.
